svelte/components/Nft/Nft.svelte

```diff
@@ -3,248 +3,229 @@
-    nftName,
-    nftOpenSeaUrl,
-    addressSame,
-    textShort,
     explorerAddressLink,
     kredeumNftUrl,
-    getNetwork
+    getNetwork,
+    nftOpenSeaUrl,
+    addressSame
   } from "lib/kconfig";
-  // import { nftKey } from "lib/kconfig";
-  import { nftGetImageLink } from "lib/knft-get-metadata";
-  import { onMount } from "svelte";
+
+  import { shortcode } from "helpers/shortcodes";
 
   import { nftStore } from "stores/nft/nft";
+
   import NftTransfer from "./NftTransfer.svelte";
 
-  /////////////////////////////////////////////////
-  //  <Nft {chainId} {address} {tokenID} {account}? {index}? {more}? {platform}? />
-  // Display NFT
-  /////////////////////////////////////////////////
   export let chainId: number;
   export let address: string;
   export let tokenID: string;
   export let account: string = undefined;
-  export let index = 0;
-  export let more = 0;
   export let platform: string = "dapp";
-
   let nft: Readable<NftType>;
 
   // let i = 1;
   // HANDLE CHANGE : on truthy chainId and address, and whatever account
   $: account, chainId && address && tokenID && handleChange();
-  const handleChange = (): void => {
-    // console.log(`NFT CHANGE #${i++} ${nftKey(chainId, address, tokenID)}`);
+  const handleChange = async (): Promise<void> => {
+    // console.log(`NFTDETAIL CHANGE #${i++} ${nftKey(chainId, address, tokenID)}`);
 
     // STATE VIEW : sync get Nft
     nft = nftStore.getOneStore(chainId, address, tokenID);
 
     // ACTION : async refresh Nft
-    nftStore.refreshOne(chainId, address, tokenID).catch(console.error);
-  };
-
-  const moreToggle = (): void => {
-    more = more > 0 ? 0 : (document.getElementById(`more-detail-${index}`)?.offsetHeight || 0) + 70;
-  };
-
-  const shortcode = async (_nft: NftType) => {
-    const data = `[kredeum_sell chain="${_nft.chainName}" collection="${_nft.address}" tokenid="${
-      _nft.tokenID
-    }" ipfs="${_nft.ipfs}"]${nftName(_nft)}[/kredeum_sell]`;
-
-    await navigator.clipboard.writeText(data).catch(() => console.log("Not copied"));
-    console.log("Copied");
-  };
-
-  const divMediaImage = (src: string, height?: number) => {
-    const heightString = height ? `height="${height}"` : "";
-    return `<img alt="link" src=${src} ${heightString}/>`;
+    await nftStore.refreshOne(chainId, address, tokenID);
   };
 
-  const divMediaVideo = (src: string, small = true) => {
-    let video: string;
-    if (small) {
-      video = '<video preload="metadata" style="border-radius: initial;">';
-    } else {
-      video =
-        '<video autoplay="true"  controls="" controlslist="nodownload" loop="" playsinline="" preload="metadata" style="border-radius: initial;">';
-    }
-    video += `<source src="${src}" type="video/mp4"></video>`;
-    return video;
+  const backToCollection = () => {
+    tokenID = "";
+    // mainContentDisplayComponent = "grid";
   };
-
-  const divMedia = (_nft: NftType, index: number, small = false) => {
```

Comment:
divmediaXXX code is duplicated in 2 components, must be only defined once,  should be there or in a helper

Reply:
divMedia fuction in helpers/mediasDisplay.ts
add one optional parameter : display = list or grid

Reply:
auto reformating problem in mediasDisplay.ts